trunion/validators.py

```diff
@@ -45,13 +46,17 @@ def valid_receipt(request):
     if receipt['iss'] not in request.registry.settings['trunion.permitted_issuers']:
         raise HTTPConflict("Bad issuer: \"%s\"" % receipt['iss'])
     if receipt['nbf'] < signing['iat']:
-        raise HTTPConflict("nbf(not before) of receipt < iat(issued at) of signing cert")
+        raise HTTPConflict("nbf(not before) of receipt < iat(issued at) of "
+                           "signing cert")
     if receipt['nbf'] > signing['exp']:
-        raise HTTPConflict("nbf(not before) of receipt > exp(expires at) of signing cert")
+        raise HTTPConflict("nbf(not before) of receipt > exp(expires at) of "
+                           "signing cert")
     if receipt['iat'] < signing['iat']:
-        raise HTTPConflict("iat(issued at) of receipt < iat(issued at) of signing cert")
+        raise HTTPConflict("iat(issued at) of receipt < iat(issued at) of "
+                           "signing cert")
     if receipt['iat'] > signing['exp']:
-        raise HTTPConflict("iat(issued at) of receipt > exp(expires at) of signing cert")
+        raise HTTPConflict("iat(issued at) of receipt > exp(expires at) of "
+                           "signing cert")
```

Comment:
in Marketplace we try to use single quotes not double quotes by default... unless the string contains a ', then a " is cool